lib/order.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/menu'
+
+class Order
+
+  attr_reader :basket, :menu, :total
+
+  def initialize(menu_klass= Menu)
+    @menu_klass = menu_klass
+    @menu = @menu_klass.new
+    @basket = Hash.new(0)
+    @total = 0
+  end
+
+  def add_order(name, quantity)
+    @basket[name] += quantity
+    basket.each {|dish, qty| "you have added #{dish} x #{qty} in your basket"}
+    @basket
+  end
+
+  def basket_sum
+    raise 'Basket is empty' if basket.empty?
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)